notes/stories/film/film-scifi-2010s.st.txt

```diff
@@ -1602,6 +1602,46 @@ polygamy [Immortal Joe's five wives fled from him.]
 the afterlife [Immortal Joe's fanatical followers were eager to die in battle so they could go to Valhalla.]
 
 
+movie: Godzilla (2014)
+======================
+
+:: Title
+Godzilla
+
+:: Description
+Godzilla is a 2014 American monster film directed by Gareth Edwards. It is a
+reboot of Toho's Godzilla franchise and is the 30th film in the Godzilla
+franchise, the first film in Legendary's MonsterVerse, and the second Godzilla
+film to be completely produced by a Hollywood studio.
+
+Synopsis: A soldier attempts to return to his family while caught in the
+crossfire of an ancient rivalry between Godzilla and two parasitic monsters
+known as MUTOs.
+
+:: Date
+2014-05-08
+
+:: References
+https://en.wikipedia.org/wiki/Godzilla_(2014_film)
+
+:: Ratings
+2 <paul>
+
+:: Major Themes
+giant monster [The titular prehistoric alpha predator Godzilla was rampaging around. Two giant insect-like creatures, dubbed Massive Unidentified Terrestrial Organisms (MUTOs), were rampaging around.]
+human vs. monster [Pandemonium swept the globe as humanity became caught in the crossfire of an ancient rivalry between Godzilla and two insect-like monsters known as MUTOs.]
+government secrecy [Much was made of the various governments of the world having kept the existence of Godzilla and the MUTO monsters secret from the public. It was revealed that the nuclear tests of the 1950s were actually secret attempts to kill Godzilla. That a multinational coalition had been formed to search for and study Godzilla was being kept secret from the public. The Japanese government covered up the MUTO attack, telling the press it was a 6.3 magnitude earthquake.]
+father and son [One subplot follows the sometimes turbulent relationship between Ford Brody and his recluse father Joe Brody. Ford Brody and his young son Sammy.]
+husband and wife [The hero of the story Ford Brody and his wife Elle worried for one another as Godzilla and the MUTOs rampaged around the world. Joe Brody and his wife Sandra we at the heart of the first part of the film.]
+nature's revenge [Humanity's discovery and widespread use of nuclear power awoke long dormant monsters (i.e. Godzilla and the MUTOs) which subsequently rampaged across various cities around the world.]
```

Comment:
Since you like to object to the subjectivity inherent in certain themes you do not like, I might just take the opportunity to point out that this here theme too is entirely subjective. Why is not every man-made pandemic, for example, a case of nature's revenge?

Reply:
I concur this theme as presently defined is problematic. For the moment, I updated the motivation for this particular usage to include how the point was spelled out in the film. Keep it here and review the usages all together in a list?